python/lambdas/copy-files-from-tdr/copy_files_lambda_function_test.py

```diff
@@ -203,7 +204,8 @@ def selective_error_object(Bucket, Key):
             str(ex.exception))
 
     def test_should_raise_an_exception_if_fields_are_missing(self):
-        with open("metadata-schema.json", "r") as metadata_schema_file:
+        schema_location = "python/lambdas/copy-files-from-tdr/metadata-schema.json"
+        with open(schema_location, "r") as metadata_schema_file:
```

Comment:
Should this be global since we are doing it twice?

Reply:
As above.